libs/deps/ost-wrapper.js

```diff
@@ -0,0 +1,26 @@
+import { loadScript, getConfig } from '../utils/utils.js';
+
+const { miloLibs, codeRoot, env } = getConfig();
+const base = miloLibs || codeRoot;
+
+export const initOst = async (el) => {
+  if (!window.ost) {
+    await loadScript(`${base}/deps/offer-selector-tool.js`);
+  }
+
+  const OstCountry = 'US';
+  const OstWcsBaseUrl = 'https://wcs.adobe.io';
+  const OstWcsApiKey = 'wcms-commerce-ims-ro-user-cc';
+  const OstAosApiKey = 'wcms-commerce-ims-user-prod';
+  const OsLanguage = 'en';
+  const OstAosAccessToken = localStorage.getItem('AOS_ACCESS_TOKEN');
```

Comment:
Do any of these need to be configurable?

Reply:
Yes, all of them. @vhargrave is working on that piece. For now, passing default temp values.